pkgs/applications/networking/irc/quassel/default.nix

```diff
@@ -31,7 +32,7 @@ assert !buildClient -> !withKDE; # KDE is used by the client only
 let
   edf = flag: feature: [("-D" + feature + (if flag then "=ON" else "=OFF"))];
 
-in with stdenv; mkDerivation rec {
+in (if daemon && !monolithic then stdenv.mkDerivation else mkDerivation) rec {
```

Comment:
I think the recommendation here was to only do this if there's like an optional qt feature, so what we want is `dontWrapQtApps = $condition` since we only care about wrapping.